include/swift/SwiftRemoteMirror/SwiftRemoteMirrorTypes.h

```diff
@@ -48,11 +48,22 @@ typedef struct swift_reflection_section {
   void *End;
 } swift_reflection_section_t;
 
+/// Represents the remote address and size of an image.
+typedef struct swift_remote_reflection_section {
+    uintptr_t StartAddress;
+    uintptr_t Size;
+} swift_remote_reflection_section_t;
+
 typedef struct swift_reflection_section_pair {
   swift_reflection_section_t section;
   swift_reflection_ptr_t offset; ///< DEPRECATED. Must be zero
 } swift_reflection_section_pair_t;
 
+typedef struct swift_local_and_remote_reflection_section {
```

Comment:
Again, Im not sure what `section` here really refers to.  Given the type, I believe that this is really the bidirectional mapping from local to remote?  Why not something like `swift_remote_reflection_metadata_mapping`?

Reply:
I used `section` as this was the existing nomenclature I found. This matches with what's in `SwiftReflectionTest` (a `ReflectionInfo` contains many `Section`s) and the exisiting `swift_reflection_section` as well. Is the name "section" wrong here? If not, what about `swift_reflection_section_mapping`, would that be better?

Reply:
Yeah, I don't think that it really is correct to encode the details of a specific file format into the name.  The nomenclature in the `ReflectionInfo` stems from it being specific to ELF.  If we want this to be limited to ELF, I think that we should be putting ELF into the name of the types (assuming that we will want custom types for MachO, WASM, COFF).  I think that the types here are pretty generic though.  I think that something like `swift_reflection_address_mapping` works well as this maps the local address to the remote address.